spec/controllers/vm_cloud_controller_spec.rb

```diff
@@ -56,6 +61,33 @@
       expect(response.body).to_not be_empty
     end
 
+    it 'can open instance resize tab' do
+      post :explorer
+      expect(response.status).to eq(200)
+
+      post :x_button, :params => {:pressed => 'vm_protect', :id => vm_openstack.id}
```

Comment:
@mansam test for vm_protect left in this block as a copy/paste error. test for 'vm_protect' button should live in a separate test block or the title of this block should be adjusted accordingly.